examples/problems/1100_callback.ipynb

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
<p>add a header above as you do for the spatial normalisation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>